Phillip Craig bumped this dev-dependency with no addtiional changes
upstream in fc0f70212e2c897ecffd7b48705783e3ebadc14b

--- rust-gimli-0.19.0/Cargo.toml
+++ rust-gimli-0.19.0/Cargo.toml
@@ -43,7 +43,7 @@
 version = "1.1.0"
 default-features = false
 [dev-dependencies.crossbeam]
-version = "0.7.1"
+version = "0.8"
 
 [dev-dependencies.getopts]
 version = "0.2"
